membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -199,7 +199,7 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
 
   def membership = lookup("membership", onSuccessMember = membershipAttributesFromAttributes, onSuccessMemberAndOrContributor = _ => notAMember, onNotFound = notFound, endpointEligibleForTest = true)
   def attributes = lookup("attributes", onSuccessMember = identity[Attributes], onSuccessMemberAndOrContributor = identity[Attributes], onNotFound = notFound, endpointEligibleForTest = true)
-  def features = lookup("features", onSuccessMember = Features.fromAttributes, onSuccessMemberAndOrContributor = _ => Features.unauthenticated, onNotFound = Features.unauthenticated, endpointEligibleForTest = true)
+  def features = lookup("features", onSuccessMember = Features.fromAttributes, onSuccessMemberAndOrContributor = Features.notAMember, onNotFound = Features.unauthenticated, endpointEligibleForTest = true)
```

Comment:
My only concern is if we double check with everybody who uses this endpoint can handle this new response.

Reply:
I'd hope they wouldn't fall apart if a new element is present in the unauthenticated response :-/

Reply:
Before, the /features response for a contributor was always:
```
{
  "adFree": false,
  "adblockMessage": true
}
```

And now it's just possible for adFree to be true